src/handlers/rfqm_handlers.ts

```diff
@@ -0,0 +1,63 @@
+// tslint:disable:max-file-line-count
+import { BigNumber } from '@0x/utils';
+import * as express from 'express';
+import * as HttpStatus from 'http-status-codes';
+
+import { CHAIN_ID } from '../config';
+import { schemas } from '../schemas';
+import { RfqmService } from '../services/rfqm_service';
+import { FetchIndicativeQuoteParams } from '../types';
+import { schemaUtils } from '../utils/schema_utils';
+import { findTokenAddressOrThrowApiError, isETHSymbolOrAddress } from '../utils/token_metadata_utils';
+
+export class RfqmHandlers {
+    constructor(public readonly _rfqmService: RfqmService) {}
+
+    public async getIndicativeQuoteAsync(req: express.Request, res: express.Response): Promise<void> {
+        const params = parseFetchIndicativeQuoteParams(req);
+        const indicativeQuote = await this._rfqmService.fetchIndicativeQuoteAsync(params);
```

Comment:
If an exception is raised here, how will it bubble up to the user?

Reply:
the catching and rethrowing logic goes here